ga4gh/frontend.py

```diff
@@ -164,6 +170,37 @@ def getReferenceSets(self):
         return app.backend.getReferenceSets()
 
 
+@app.before_first_request
+def configure_oid():
```

Comment:
It would be good to have a docstring comment here to describe what this method does. Also, it would be good to clarify why this needs to be done before the first request, and not at configure time.